### ISSUE TITLE
Typing too much into the console causes DF to crash.

### ISSUE THREAD

**Issue by keupo** [DFHack/dfhack#1354]
Using the 0.44.12-alpha1 64-bit mac version.

Typing too long of a command, 32 characters, into the console causes dfhack and the game to crash.

The in-game command line from ctrl+shift+p will accept these longer commands and serves as a workaround.

**Comment by lethosor** [comment]
I've never had this happen. Do you know what you were typing? I'm assuming 0.44.12, in any case.

**Comment by lethosor** [comment]
```
DFHack version 0.44.12-alpha1 (release) on x86_64
Type in '?' or 'help' for general help, 'ls' to see all commands.
[DFHack]# 1234567890qwertyuiopasdfghjklzxcvbnm1234567890qwertyuiopasdfghjklzxcvbnm
1234567890qwertyuiopasdfghjklzxcvbnm1234567890qwertyuiopasdfghjklzxcvbnm is not a recognized command.
```
Can't reproduce this

**Comment by lethosor** [comment]
Ok, looks like #1355 addresses this (I had to enter ~90 characters - your terminal window must be pretty small).

**Comment by lethosor** [comment]
Fixed in #1355
